widget/embedded/src/components/SettingList.tsx

```diff
@@ -0,0 +1,266 @@
+import type { ListPropTypes } from '@rango-dev/ui/dist/widget/ui/src/components/List/List.types';
+import type { SvgIconProps } from '@rango-dev/ui/dist/widget/ui/src/components/SvgIcon';
+
+import { i18n } from '@lingui/core';
+import {
+  AutoThemeIcon,
+  BridgeIcon,
+  ChevronRightIcon,
+  DarkModeIcon,
+  Divider,
+  DoneIcon,
+  ExchangeIcon,
+  InfoIcon,
+  LanguageIcon,
+  LightModeIcon,
+  List,
+  ListItem,
+  ListItemButton,
+  Skeleton,
+  styled,
+  Switch,
+  Tabs,
+  Tooltip,
+  Typography,
+} from '@rango-dev/ui';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { SlippageTooltipContainer as TooltipContainer } from '../components/Slippage/Slippage.styles';
+import { navigationRoutes } from '../constants/navigationRoutes';
+import { ThemeMode } from '../constants/settings';
+import { useAppStore } from '../store/AppStore';
+import { getContainer } from '../utils/common';
+import { getUniqueSwappersGroups, isFeatureHidden } from '../utils/settings';
+
+const ThemeSection = styled('div', {
+  width: '202px',
+  height: '$40',
+});
+
+const themesList = [
+  {
+    id: ThemeMode.LIGHT,
+    icon: <LightModeIcon color="black" size={16} />,
+    tooltip: (
+      <Typography size="xsmall" variant="body">
+        {i18n.t('Light')}
+      </Typography>
+    ),
+  },
+  {
+    id: ThemeMode.DARK,
+    icon: <DarkModeIcon color="black" size={16} />,
+    tooltip: (
+      <Typography size="xsmall" variant="body">
+        {i18n.t('Dark')}
+      </Typography>
+    ),
+  },
+  {
+    id: ThemeMode.AUTO,
+    icon: <AutoThemeIcon color="black" size={16} />,
+    tooltip: (
+      <Typography size="xsmall" variant="body">
+        {i18n.t('Auto')}
+      </Typography>
+    ),
+  },
+];
+
+const getThemeIcon = (theme: ThemeMode) => {
+  const iconProps: SvgIconProps = { color: 'gray', size: 12 };
+
+  switch (theme) {
+    case 'auto':
+      return <AutoThemeIcon {...iconProps} />;
+    case 'dark':
+      return <DarkModeIcon {...iconProps} />;
+    default:
+      return <LightModeIcon {...iconProps} />;
+  }
+};
+
+export function SettingList() {
```

Comment:
I think it's better to rename it to SettingsLists (also rename file)
and move this file to SettingsContainer maybe? 🤔

Reply:
yes it's better . Done